### DISCUSSION TITLE
GitHub Classroom UI Customization - is it possible?

### DISCUSSION THREAD

**Discussion by cybereducenter** [community/community#102797] [GitHub Education]
### Hi everyone,

We are new to GitHub Classroom and currently in design stages of our Organizational courses.
Is the UI and layout of GitHub Classroom customizable to better match our specific organizational needs? Can I change terminology or add custom branding elements, change views etc...?

Thank you !

**Comment by phuccoder2000** [comment]
However, the extent of customization is somewhat restricted.

As of the last update, some of the customization options include:
1. Assignment Naming:
- In GitHub Classroom, you can customize the names of your assignments.
2. Readme Files:
- You can include custom instructions or information in the README.md files associated with your assignments or repositories.
3. GitHub Pages:
- You can utilize GitHub Pages to create a more customized landing page for your organization, course, or assignments. This allows you to provide additional information and resources.
4. GitHub Repository Settings:
- You can customize the settings of individual repositories, including permissions, collaborators, and other repository-specific configurations.

However, as of the last update, there were limitations regarding broader UI/UX customization, terminology changes, and branding elements. GitHub Classroom tends to follow GitHub's standard interface for consistency across the platform.
